corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -9,14 +9,38 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
         template: _.template($("#query-view-item-template").html() || ""),
 
         templateContext: function () {
-            var imageUri = this.options.model.get('imageUri');
-            var audioUri = this.options.model.get('audioUri');
-            var appId = this.model.collection.appId;
+            var imageUri = this.options.model.get('imageUri'),
+                audioUri = this.options.model.get('audioUri'),
+                appId = this.model.collection.appId,
+                initialValue = this.options.model.get('value');
+
+            // Initial values are sent from formplayer as strings, but dropdowns expect an integer
+            if (initialValue && this.options.model.get('input') === "select1") {
+                initialValue = parseInt(initialValue);
+            }
+
             return {
```

Comment:
After merging with this change, I noticed an issue where this.options.model.get('input') was not defined, which caused a ReferenceError for the use of 'input' in query-view-list-template. Inserting a dummy value into the Backbone model caused the error to go away.

It might be relevant that the case list I was trying to query was defined before I merged, so it wasn't written to the database with the latest code. However, I did run the migrate command before testing.

Reply:
Thank you for flagging. Have you pulled formplayer recently? This PR depends on some formplayer code, also just merged today, and the behavior you're describing sounds like formplayer is on an old commit that isn't passing `input` along.

Reply:
Ah, that was it. It works fine after fetching a new formplayer. Thanks for the quick answer!